SimPEG/utils/model_utils.py

```diff
@@ -100,42 +100,64 @@ def ismember(a, b):
 def depth_weighting(
     mesh, reference_locs, active_cells=None, exponent=2.0, threshold=None, **kwargs
 ):
-    """A simple depth weighting function
+    r"""
+    Construct diagonal elements of a depth weighting matrix
 
-    This function is a simple form of depth weighting based off of the vertical distance
-    of mesh cell centers from the reference location(s).
-
-    This is commonly used to counteract the natural decay of potential field data at
-    depth.
+    Builds the model weights following the depth weighting strategy, a method
+    to generate weights based on the vertical distance between mesh cell
+    centers and some reference location(s).
+    Use these weights in regularizations to counteract the natural decay of
+    potential field data with depth.
 
     Parameters
     ----------
-    mesh : discretize.base.BaseMesh
-        discretize model space.
-    reference_locs : float or (n, dim) numpy.ndarray
-        the reference values for top of the points
-    active_cells : (mesh.n_cells) numpy.ndarray of bool, optional
-        index vector for the active cells on the mesh.
-        A value of ``None`` implies every cell is active.
+    mesh : :class:`discretize.base.BaseMesh`
```

Comment:
you don't need the :class: or :meth: rst decorators here for numpy docstrings. It will automatically put those in for you.